app/views/home/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<% if logged_in? %>
+        <div>
+                <h1> Home </h1>
+                <p> <strong> Welcome </strong> <%= current_user.name %> </p>
+                <p> <a href = "<%= user_path(current_user) %>"> My Account </a> </p>
+                <p> <a href = "logout"> Logout </a> </p>
+        </div>
+<% else %>
+        <h1>Welcome To Awesome Blog</h1>
+        <div> 
+                <strong> New user ! </strong> 
+                <a href = 'signup'> Sign up here</a> 
+        </div>
+        <div> 
+                <strong> Already have an account. </strong>
+                <a href = 'login'> Log in here </a> 
+        </div>
+        <br><br>
+<% end %>
+
+Our site has <%= Article.public_count %> articles and counting!
+<ul>
```

Comment:
1. What if there are 1 lakh articles?
2. How many queries will this make?
3. Why fetch private articles from db when we don't need to show them?